Package.swift

```diff
@@ -66,44 +66,6 @@ automatic linking type with `-auto` suffix appended to product's name.
 */
 let autoProducts = [swiftPMProduct, swiftPMDataModelProduct]
 
-var packageCollectionsSigningTargets = [Target]()
-var packageCollectionsSigningDeps: [Target.Dependency] = [
-    "Basics",
-    .product(name: "Crypto", package: "swift-crypto"),
-    "PackageCollectionsModel",
-]
-// swift-crypto's Crypto module depends on CCryptoBoringSSL on these platforms only
-#if os(Linux) || os(Windows) || os(Android)
-packageCollectionsSigningTargets.append(
-    .target(
-        /** Package collections signing C lib */
-        name: "PackageCollectionsSigningLibc",
-        dependencies: [
-            .product(name: "Crypto", package: "swift-crypto"), // for CCryptoBoringSSL
```

Comment:
done still need the dependency on swift-crypto?

Reply:
Yes, we still need it for keys and signing.